source/fundamentals/run-command.txt

```diff
@@ -55,17 +55,46 @@ the current member's role in the replica set, on a database:
 For a full list of database commands and corresponding parameters, see
 the :ref:`Additional Information section <addl-info-runcommand>`.
 
+.. _node-command-options:
+
+Command Options
+---------------
+
+You can set options when executing a database command by
+passing an options object to either command method. The ``command()`` method
+takes a ``RunCommandOptions`` object, and the ``runCursorCommand()`` method
+takes a ``RunCursorCommandOptions`` object.
+
+Starting in version 6.0 of the {+driver-short+}, you can pass only the
+following options directly to the ``command()`` method:
+
+- ``comment``
+- ``enableUtf8Validation``
+- ``raw``
+- ``readPreference``
+- ``session``
+
+You can set command-specific options, including a write or read
+concern, in the command document that you pass to ``command()``. The
```

Comment:
Suggestion:

1) I think the reader would benefit from a clear differentiation between the command options described in the first paragraph and "command-specific options".

I also think that it might not be clear which options can be specified in the command document, and whether it depends on the command.

To contrast this with the "only the following options" from the prior paragraph:
"You can specify additional commands on certain commands..." 


2) I think the mention of write or read concerns as options that you can specify in the command document, followed by an example of maxTimeMS is confusing. To resolve this, provide the reader with a reference that they can check to determine which options they can provide which commands.